test/CosmosDB/Microsoft.Azure.Workflows.ServiceProvider.Extensions.CosmosDB.Tests/CosmosDBOperationsTests.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Workflows.ServiceProvider.Extensions.CosmosDB.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Cosmos;
+    using Microsoft.Azure.WebJobs.Host.TestCommon;
+    using Microsoft.Azure.Workflows.ServiceProviders.Abstractions;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.WindowsAzure.ResourceStack.Common.Json;
+    using Moq;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    /// <summary>
+    /// Cosmos DB End2End tests.
+    /// </summary>
+    public class CosmosDBOperationsTests
+    {
+        /// <summary>
+        /// output logger for test.
+        /// </summary>
+        private readonly ITestOutputHelper outputLogger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CosmosDBOperationsTests"/> class.
+        /// </summary>
+        /// <param name="output">Output</param>
+        public CosmosDBOperationsTests(ITestOutputHelper output)
+        {
+            this.outputLogger = output;
+        }
+
+        /// <summary>
+        /// Operation connection parameters test.
+        /// </summary>
+        [Fact]
+        public void OperationConnectionParametersTest()
```

Comment:
we should probably add more of an end to end test to make sure all the extension initialization paths works as well (e.g., run this on top of Functions runtime and make sure apis are loaded)

Reply:
added webjobs loading and API level E2E testing